pkg/cpuid/cpuid_amd64.go

```diff
@@ -322,6 +322,10 @@ func (fs FeatureSet) WriteCPUInfoTo(cpu, numCPU uint, w io.Writer) {
 	fmt.Fprintf(w, "model name\t: %s\n", "unknown") // Unknown for now.
 	fmt.Fprintf(w, "stepping\t: %s\n", "unknown")   // Unknown for now.
 	fmt.Fprintf(w, "cpu MHz\t\t: %.3f\n", cpuFreqMHz)
+	// Pretend the CPU has 512 KB of cache. Note that real /proc/cpuinfo exposes total L3 cache
+	// size on Intel and per-core L2 cache size on AMD (as of Linux 6.1.0), so the value of this
+	// field is not really important in practice.
+	fmt.Fprintf(w, "cache size\t: 512 KB\n")
```

Comment:
Can you still use a value closer to the likely true value? On the two regular consumer-level computers I've checked so far, this is between 11264 KiB and 24576 KiB. So 512 KiB feels like the wrong order of magnitude.

Reply:
@EtiennePerot Thanks for comment. The value is not actually wrong - it comes directly from an AMD machine which has 512KB of L2 RAM per core. All AMD machines will show somewhere between 256 and 512 KB of `cache size`. On Intel `cache size` reports last level cache, so we will see tens and hundreds megabytes here.

This is the problem with `cache size` value - it's not very useful if it's not consistent.

Reply:
I updated the PR to use 8MB as the value reported here. Justification has been added as a comment:

```
// 8192 KB is selected because it is a reasonable size that will be effectively usable on
// lightly loaded machines - most machines have 1-4MB of L3 cache per core.
```